core/src/main/java/nl/nn/adapterframework/http/rest/ApiServiceDispatcher.java

```diff
@@ -232,11 +233,11 @@ private JsonArrayBuilder mapServers(HttpServletRequest request) {
 		String loadBalancerUrl = AppConstants.getInstance().getProperty("loadBalancer.url", null);
 		if(StringUtils.isNotEmpty(loadBalancerUrl)) {
 			serversArray.add(Json.createObjectBuilder().add("url", loadBalancerUrl + servletPath).add("description", "load balancer"));
-		}
-		else { // fall back to the request url
+		} else { // fall back to the request url
 			String requestUrl = request.getRequestURL().toString(); // -> schema+hostname+port/context-path/servlet-path/+request-uri
+			requestUrl = Misc.urlDecode(requestUrl); // request url from browser might be encoded
```

Comment:
I assume this solves the problem? It would be wise to include a test using an encoded requestUrl